src/components/MovieForm/MovieForm.tsx

```diff
@@ -0,0 +1,152 @@
+                />
+            </div>
+
+            <div className={styles.labelAndInputContainer}>
+                <label className={styles.inputLabel}
+                       htmlFor="rating">Rating</label>
+                <input className={styles.input}
+                       type="number"
+                       id="rating"
+                       name="rating"
+                       aria-label="Movie Rating"
+                       defaultValue={initialMovieInfo?.rating || ""}
+                       placeholder="7.8"
+                       required={true}
+                />
+            </div>
+
+            <div className={styles.labelAndInputContainer}>
+                <label className={styles.inputLabel}
+                       htmlFor="genres">Genre</label>
+                <select className={styles.input}
+                        id="genres"
+                        name="genres"
+                        aria-label="Movie Genres"
+                        defaultValue={initialMovieInfo?.genres || [""]}
+                        multiple={true}
+                        required={true}
+                >
+                    <option value="Action">Action</option>
+                    <option value="Crime">Crime</option>
+                    <option value="Documentary">Documentary</option>
+                    <option value="Thriller">Thriller</option>
+                    <option value="Musical">Musical</option>
+                </select>
+            </div>
+
+            <div className={styles.labelAndInputContainer}>
+                <label className={styles.inputLabel}
+                       htmlFor="runtime">Runtime</label>
+                <input className={styles.input}
+                       type="text"
+                       id="runtime"
+                       name="runtime"
+                       aria-label="Movie Runtime"
+                       defaultValue={initialMovieInfo?.runtime || ""}
+                       placeholder="minutes"
+                       required={true}
+                />
+            </div>
+
+            <div className={styles.overviewContainer}>
+                <label className={styles.inputLabel}
+                       htmlFor="description">Overview</label>
+                <textarea className={styles.input}
+                          id="description"
+                          name="description"
+                          aria-label="Movie Description"
+                          defaultValue={initialMovieInfo?.description || ""}
+                          placeholder="Movie description"
+                />
+            </div>
+
+            <div className={styles.buttonsContainer}>
+                <button className={styles.resetButton}
+                        type="reset">Reset</button>
+                <button className={styles.submitButton}
+                        type="submit"
+                >
+                    Submit</button>
+            </div>
+        </form>
```

Comment:
Also like two components 
![image](https://github.com/user-attachments/assets/1b92fc1e-8fad-4f62-83f2-aad2a059f0ce)

React - for crating component and reuse it or just separate different things to make it more readable.

Reply:
I created two components to move out some logic and markup to us in the MovieForm: InputField and GenreSelect.
Can we leave the movie description and these two buttons as it is? I might move them out if they will be used in other places